action.sh

```diff
@@ -194,6 +216,8 @@ systemctl enable shutdown.service
     ./svc.sh install && \\
     ./svc.sh start && \\
     gcloud compute instances add-labels ${VM_ID} --zone=${machine_zone} --labels=gh_ready=1
+    # Install at if not installed
+    command -v at >/dev/null 2>&1 ||  apt-get install -y -q at
```

Comment:
#31 has removed the need for `at`

Reply:
Removed in [945578f](https://github.com/related-sciences/gce-github-runner/pull/33/commits/945578fb8561353f87bb9984fdee7cc73f803a78)